app/apphandlers/embeddedAppHandler.go

```diff
@@ -243,7 +256,7 @@ func (a *embeddedApp) StartRudderCore(ctx context.Context, options *app.Options)
 		GatewayDB:       gwDBForProcessor,
 		RouterDB:        routerDB,
 		BatchRouterDB:   batchRouterDB,
-		ErrorDB:         errDB,
+		ErrorDB:         errDBForRead,
```

Comment:
@cisse21 does migrator migrate jobs from `proc_errors`? I assume not, correct?

Reply:
Yes migrator does migrate jobs from proc_errors

Reply:
@chandumlg maybe we need another jobsdb for gw errors

Reply:
Let me know if this doesn't make sense. We can discuss this is over a call

Reply:
so no issue?